Update content for SOI trainings

ClaireMcKayBowen opened the pull request:
Updated the content and made the lesson into a Quarto document.

ClaireMcKayBowen committed:
Update content for SOI trainings

Updated the content and made the lesson into a Quarto document.

mclaire19 commented on lessons/03_introduction_to_differential_privacy.qmd:
```diff
@@ -0,0 +1,550 @@
+
+In most of the cases we've discussed so far, the released data product is a full dataset. However, a spectrum of data products could be released by a data curator after applying privacy methods. Here are a list examples of possible data products that a data curator could release after applying SDC methods, roughly from most to least detailed:
+
+- mmicrodata (e.g., public use microdata series or PUMS) 
```
*microdata (typo)

mclaire19 commented on lessons/03_introduction_to_differential_privacy.qmd:
```diff
@@ -0,0 +1,550 @@
+
+- If too many questions are answered with enough accuracy, all the data could be compromised, so the type and number of questions asked of the data are limited by the curators [@bowen2021philosophy].
+
+The main difference between traditional SDC methods and formally private methods is the ability to account for all information being "leaked" from the confidential data. We can think of traditional SDC methods as someone charging a limitless credit card; formally private methods are when someone charges to a debit card with a set budget. In both scenarios, there is a running bill, but only one re ...
```
I will always love this metaphor!

mclaire19 commented on lessons/03_introduction_to_differential_privacy.qmd:
```diff
@@ -0,0 +1,550 @@
+
+In this exercise, we're going to practice applying the Laplace mechanism by drawing from the Laplace distribution using varying values of sensitivity and epsilon.
+
+Using the `laplace_mechanism()` function defined above (make sure you have run this code), vary the `epsilon` and `sensitivity` parameters. Some sample code is included below. Since we are not trying to reproduce these results, there is no need to set a seed.
```
I don't actually see this defined anywhere. If we're just trying to show the intuition without coding (which I think we should aim for given that we haven't run code so far in this series), should we change up the phrasing?

ClaireMcKayBowen committed:
Maddie's comments and Laplace

Addressed Maddie's comments, modified exercise 2, added images of laplace distribution.
